tests/test_schema.py

```diff
@@ -351,6 +352,18 @@ class Model(BaseModel):
     }
 
 
+def test_strict_bool():
```

Comment:
This is the test for schema. Is something else needed?

Reply:
yes, sorry.